arch/wasm32/wasm.js

```diff
@@ -821,66 +825,70 @@ var musl_hack = (function() {
-    fegetround: NYI('fegetround'),
-    feraiseexcept: NYI('feraiseexcept'),
-    fesetround: NYI('fesetround'),
-    fetestexcept: NYI('fetestexcept'),
     __fixtfdi: NYI('__fixtfdi'),
     __fixtfsi: NYI('__fixtfsi'),
-    __floatscan: NYI('__floatscan'),
+    __fixunstfsi: NYI('__fixunstfsi'),
     __floatsitf: NYI('__floatsitf'),
-    fprintf: NYI('fprintf'),
-    iswctype: NYI('iswctype'),
-    iswspace: NYI('iswspace'),
-    __lctrans: NYI('__lctrans'),
-    __lctrans_cur: NYI('__lctrans_cur'),
+    __floatunsitf: NYI('__floatunsitf'),
+    __getf2: NYI('__getf2'),
+    getrlimit: NYI('getrlimit'),
+    __gttf2: NYI('__gttf2'),
+    ioctl: NYI('ioctl'),
+    __letf2: NYI('__letf2'),
     __lock: NYI('__lock'),
-    __lockfile: NYI('__lockfile'),
+    longjmp: NYI('longjmp'),
     __lttf2: NYI('__lttf2'),
-    __madvise: NYI('__madvise'),
-    mbtowc: NYI('mbtowc'),
-    __mmap: NYI('__mmap'),
-    __mremap: NYI('__mremap'),
+    __muldc3: NYI('__muldc3'),
+    __mulsc3: NYI('__mulsc3'),
+    __multc3: NYI('__multc3'),
     __multf3: NYI('__multf3'),
-    __munmap: NYI('__munmap'),
-    nanosleep: NYI('nanosleep'),
     __netf2: NYI('__netf2'),
-    posix_spawn: NYI('posix_spawn'),
-    posix_spawn_file_actions_adddup2: NYI('posix_spawn_file_actions_adddup2'),
-    posix_spawn_file_actions_destroy: NYI('posix_spawn_file_actions_destroy'),
-    posix_spawn_file_actions_init: NYI('posix_spawn_file_actions_init'),
-    pthread_self: NYI('pthread_self'),
-    __randname: NYI('__randname'),
-    __rem_pio2_large: NYI('__rem_pio2_large'),
-    setitimer: NYI('setitimer'),
+    nftw: NYI('nftw'),
+    pthread_attr_init: NYI('pthread_attr_init'),
+    pthread_attr_setdetachstate: NYI('pthread_attr_setdetachstate'),
+    pthread_attr_setguardsize: NYI('pthread_attr_setguardsize'),
+    pthread_attr_setstacksize: NYI('pthread_attr_setstacksize'),
+    pthread_barrier_destroy: NYI('pthread_barrier_destroy'),
+    pthread_barrier_init: NYI('pthread_barrier_init'),
+    pthread_barrier_wait: NYI('pthread_barrier_wait'),
+    pthread_cancel: NYI('pthread_cancel'),
+    _pthread_cleanup_pop: NYI('_pthread_cleanup_pop'),
+    _pthread_cleanup_push: NYI('_pthread_cleanup_push'),
+    pthread_cond_init: NYI('pthread_cond_init'),
+    pthread_cond_wait: NYI('pthread_cond_wait'),
+    pthread_create: NYI('pthread_create'),
+    pthread_mutex_init: NYI('pthread_mutex_init'),
+    pthread_mutex_lock: NYI('pthread_mutex_lock'),
+    pthread_mutex_unlock: NYI('pthread_mutex_unlock'),
+    pthread_once: NYI('pthread_once'),
+    pthread_rwlock_rdlock: NYI('pthread_rwlock_rdlock'),
+    pthread_rwlock_unlock: NYI('pthread_rwlock_unlock'),
+    pthread_rwlock_wrlock: NYI('pthread_rwlock_wrlock'),
+    pthread_setcancelstate: NYI('pthread_setcancelstate'),
+    pthread_sigmask: NYI('pthread_sigmask'),
+    pthread_testcancel: NYI('pthread_testcancel'),
+    sem_init: NYI('sem_init'),
+    sem_post: NYI('sem_post'),
+    sem_wait: NYI('sem_wait'),
```

Comment:
Surprising to see sem and pthread things here, but I guess they're not implemented for threading yet? You probably don't want them in `musl_hack`.

Reply:
As I said, this is auto-generated.      I could add the pthread sources too, which would reduce this.. but not sure if we want to be including musl's pthread impl.